Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node
+WORKDIR /app
+COPY package.json .
+RUN npm i
+COPY . .
+
+ENV NETWORK=mainnet
+ENV NODE_ENV=docker
```

Comment:
Not a valid value https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production#nodejs-the-difference-between-development-and-production